redbox-core/redbox/transform.py

```diff
@@ -61,21 +65,18 @@ def flatten_document_state(documents: DocumentState) -> list[Document]:
     return [document for group in documents.values() for document in group.values()]
 
 
-def to_request_metadata(response_metadata: dict):
-    """
-    Convert a Langchain ChatLLM response metadata dictionary
+@RunnableLambda
+def to_request_metadata(output: dict):
```

Comment:
```suggestion
def to_request_metadata(output: dict) -> RequestMetadata:
```

nitpick: `output` feels like a strange variable name for an input?

Reply:
otherwise nice and clear